docs/proposals/2019-03-18-Links-List.md

```diff
@@ -0,0 +1,112 @@
+## Problem
+
+Organizations have requested a way to include sidebars, related content boxes, or other recirculation links within the body of an article. A curated solution -- where users have reasonable control over the presentation and selection of the items on an individual -- is highly-desirable.
+
+Existing solutions are either too generic (`element_group`, `story`) to interpret well in a UX, or too limited (`related_content`, `promo_items`) to be useful in this capacity, or cannot be meaningfully extended in a non-breaking way.
+
+
+## Proposal
+
+Add a new content element `links_list` that contains a narrowly-targeted set of functionality around inline recirculation boxes. Applicable to stories only.
+
+A links_list may have:
+* A plaintext title
+* A subtype (selected from an enumerated-list configured on a per-organization basis)
+* A sequence of up to 10 and at least one `links_list_item`s, each of which has:
+    * A subtype
+    * A plaintext title (required)
+    * A link (required)
+    * A description (ANS `text` element)
+    * An image (ANS `image` or `reference` to an image)
+
+
+## Example
+
+```json
+{
+  "type": "story",
+  "version": "0.10.x",
+
+  "headlines": {
+    "basic": "Man Bites Dog"
+  },
+
+  "content_elements": [
+    {
+      "type": "links_list",
+
+      "subtype": "sidebar",
+      "title": "More 'Biting' Stories",
+
+      "items": [
+        {
+          "type": "links_list_item",
+          "subtype": "blue-label",
+
+          "title": "Lorem ipsum",
+          "link": "http://www.google.com",
```

Comment:
@gengel This looks alot like "interstitial_link".  Would this be intended as a replacement, with a new subtype of "interstitial_link"?

Reply:
It's a similar use case, but not quite the same. The intention is that this is it's own, new element type for groups of links with a title for the box/section, a plaintext title for each link, optional images, but no embedded HTML or formatting. 

This is a little bit more structured, and more of an "aside" to the main content.

Reply:
We did call the link "url" over in interstitial, though, so I should probably change the field here to be called the same.